python/ray/serve/tests/test_application_state.py

```diff
@@ -216,7 +217,7 @@ def test_deploy_with_renamed_app():
     app_status = app_state_manager.get_app_status("app1")
     assert app_status.status == ApplicationStatus.DEPLOYING
 
-    app_state_manager.deployment_state_manager.set_deployment_statuses_healthy(0)
+    app_state_manager._deployment_state_manager.set_deployment_statuses_healthy(0)
```

Comment:
ugly that we need to reach into a private field (but not a huge deal)

is there a better way to structure the test?

Reply:
Stored the mock deployment state manager into a variable before passing to app state manager and used that instead in the tests - lmk if that looks better